module/VuFind/src/VuFind/Recommend/Databases.php

```diff
@@ -263,11 +274,12 @@ public function getResults()
         if ($this->useQuery) {
             $queryObject = $this->results->getParams()->getQuery();
             $query = is_callable([$queryObject, 'getString'])
-                ? strtolower($queryObject->getString())
+                ? $this->normalizeQueryString($queryObject->getString())
                 : '';
             if (strlen($query) >= $this->useQueryMinLength) {
                 foreach ($nameToDatabase as $name => $databaseInfo) {
-                    if (str_contains(strtolower($name), $query)) {
+                    $name = $this->normalizeQueryString($name);
+                    if (str_contains($name, $query)) {
```

Comment:
Is there a reason to modify the `$name` variable? It seems like it actually might be more clear to do this inline:

```suggestion
                    if (str_contains($this->normalizeQueryString($name), $query)) {
```

Reply:
Yeah good call.